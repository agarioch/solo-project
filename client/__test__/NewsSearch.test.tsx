import React from 'react';
import renderer from 'react-test-renderer';
import NewsSearch from '../components/NewsPage/NewsSearch';

test('news search renders correctly', () => {
  const tree = renderer
    .create(<NewsSearch input={''} filterNews={() => {}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
