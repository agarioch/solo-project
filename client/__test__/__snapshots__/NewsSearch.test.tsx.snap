// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`news search renders correctly 1`] = `
<RCTSafeAreaView
  accessible={true}
  emulateUnlessSupported={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "marginTop": 30,
      "padding": 40,
    }
  }
>
  <TextInput
    allowFontScaling={true}
    clearButtonMode="always"
    keyboardAppearance="dark"
    onChangeText={[Function]}
    placeholder="Search..."
    placeholderTextColor="#fff"
    rejectResponderTermination={true}
    style={
      Object {
        "borderColor": "#BFD7ED",
        "borderRadius": 30,
        "borderWidth": 2,
        "color": "#fff",
        "fontWeight": "bold",
        "letterSpacing": 1,
        "padding": 15,
        "paddingLeft": 20,
        "width": "70%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
</RCTSafeAreaView>
`;
